eth/vm/base.py

```diff
@@ -81,11 +81,11 @@
 
 
 class BaseVM(Configurable, ABC):
-    block = None  # type: BaseBlock
-    block_class = None  # type: Type[BaseBlock]
-    fork = None  # type: str
-    chaindb = None  # type: BaseChainDB
-    _state_class = None  # type: Type[BaseState]
+    block: BaseBlock = None
+    block_class: Type[BaseBlock] = None
+    fork: str = None    # noqa: E701
```

Comment:
It didn't make sense to me that `flake8` was emitting this error, so I dug in a bit more. Turns out that it's a bug that would be fixed by updating `flake8`. But they keep adding new warnings, so that would add a bunch of unrelated work. Can you add an extra comment beyond this mentioning that it's a `flake8` bug that's fixed in 3.6.0+?

(unrelated: it's more common in our code base to have just two spaces before the `#`)

Both changes together look like:
```suggestion
    fork: str = None  # noqa: E701  # flake8 bug that's fixed in 3.6.0+
```

So this should apply everywhere that there's an `E701` in the PR.

Reply:
Am running the command now. Expect the PR in 10 minutes

Reply:
> It didn't make sense to me that `flake8` was emitting this error, so I dug in a bit more. Turns out that it's a bug that would be fixed by updating `flake8`. But they keep adding new warnings, so that would add a bunch of unrelated work. Can you add an extra comment beyond this mentioning that it's a `flake8` bug that's fixed in 3.6.0+?
> 
> (unrelated: it's more common in our code base to have just two spaces before the `#`)
> 
> Both changes together look like:
> 
> So this should apply everywhere that there's an `E701` in the PR.

I've added the `flake8` comment. But this `eth/vm/base.py` has refused to get fixed.